octoprint_mrbeam/static/js/app/view-models/material-store.js

```diff
@@ -25,13 +26,13 @@ $(function () {
                     );
                 }
 
-                $.ajax({
-                    url: healthcheck_url,
-                    method: "HEAD",
-                    timeout: 6000,
-                    success: self.loadMaterialStore,
-                    error: self.showConnectionError,
-                });
+                if (healthcheck_url !== self.healthcheck_url) {
+                    self.healthcheck_url = healthcheck_url;
+                }
+
+                fetch(self.healthcheck_url)
```

Comment:
the fetch timeouts at 300 seonds on default, would it be better to limit it to a lower value?
https://dmitripavlutin.com/timeout-fetch-request/